app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/MediaControlsWidget.java

```diff
@@ -197,8 +197,13 @@ public void onSeekPreview(String aText, double aRatio) {
             if (mMedia == null) {
                 return false;
             }
+            if(event.getX()<0)
+            {
+                mVolumeControl.setVisibility(View.GONE);
+                return false;
+            }
```

Comment:
A few formatting nits:
```
if(event.getX()<0)
{
}
```
Should be:
```
if (event.getX() < 0) {
}
```
Space after `if`. Space on each side of `<`.  Curly brackets `{}` should be K&R style.